MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>DataFormRadioGroupItem</td>
+<td>Represents the class that encapsulates the layout and RadioGroup editor settings for data field.</td>
+</tr>
+
+<tr>
+<td>{{'[DataFormTextItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTextItem.html)'| markdownify }}</td>
+<td>DataFormTextItem</td>
+<td>Represents the class that encapsulates the layout and text editor setting for a data field.</td>
+</tr>
+
+<tr>
+<td>{{'[DataFormTimeItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTimeItem.html)'| markdownify }}</td>
+<td>DataFormTimeItem</td>
+<td>Represents the class that encapsulates the layout and time picker editor setting for a data field.</td>
+</tr>
+
+<tr>
+<td>{{'[DateRangeAttribute](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html)'| markdownify }}</td>
+<td>DataFormDateRangeAttribute</td>
+<td>Specifies the date range constraints for the value of a data field.</td>
+</tr>
+
+<tr>
+<td>{{'[DisplayOptionsAttribute](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html)'| markdownify }}</td>
+<td>DataFormDisplayOptionsAttribute</td>
+<td>Provides a attribute that lets you specify the display settings for a data field.</td>
+</tr>
+
+<tr>
+<td>{{'[LabelPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelPosition.html)'| markdownify }}</td>
+<td>DataFormLabelPosition</td>
+<td>Defines the constants for label position that specifies whether to layout the label at left or top of editor.</td>
+</tr>
+
+<tr>
+<td>{{'[SfDataForm](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html)'| markdownify }}</td>
+<td>SfDataForm</td>
+<td>The DataForm control is used to manage data and develop various forms such as login, reservation, data entry and so on.</td>
+</tr>
+
+<tr>
+<td>{{'[SourceProvider](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SourceProvider.html)'| markdownify }}</td>
+<td>IDataFormSourceProvider</td>
+<td>Represents the interface to provide method to get the source for picker type properties.</td>
+</tr>
+
+<tr>
+<td>{{'[LabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html)'| markdownify }}</td>
+<td>DataFormTextStyle</td>
+<td>Gets or sets properties which allows to customize the editor text style in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidatingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidatingEventArgs.html)'| markdownify }}</td>
+<td>DataFormValidateFormEventArgs</td>
+<td>Provides data for ValidateForm event.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>DataFormValidationMode</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
```

Comment:
<td>Defines the constants for the ValidationMode that specifies when the user input should be validated for the data object.</td>